staff_features/accessions/accession_edit.feature

```diff
@@ -0,0 +1,57 @@
+Feature: Accession Edit
+  Background:
+    Given an administrator user is logged in
+      And an Accession has been created
+  Scenario: Accession is opened in the edit mode from the browse menu
+    Given the Accession appears in the search results list
+     When the user clicks on 'Edit'
+     Then the Accession is opened in the edit mode
+      And all fields become editable
+  Scenario: Accession is opened in the edit mode from the view mode
+    Given the Accession is opened in the view mode
+     When the user clicks on 'Edit'
+     Then the Accession is opened in the edit mode
+      And all fields become editable
+  Scenario Outline: Accession is successfully updated
+    Given the Accession is opened in edit mode
+     When the user changes the '<Field>' field to '<NewValue>'
+      And the user clicks on 'Save'
+     Then the 'Accession' updated message is displayed
+     Then the field '<Field>' has value '<NewValue>'
+     Examples:
+      | Field    | NewValue                 |
+      | Title    | Updated Test Accession   |
+      | Accession Date     | 2024-10-03               |
+  Scenario: Accession is not updated after changes are reverted
+    Given the Accession is opened in edit mode
+     When the user changes the 'Title' field
+      And the user clicks on 'Revert Changes'
+     Then the 'Title' field has the original value
```

Comment:
see my comments on this test in the accession_edit step definitions

Reply:
Fixed.